Libraries/Components/View/View.js

```diff
@@ -258,6 +258,35 @@ const View = React.createClass({
      * Used to locate this view in end-to-end tests.
      *
      * > This disables the 'layout-only view removal' optimization for this view!
+     *
+     * ### Android Specifics
+     *
+     * By default this value is passed to the underlying View's
+     * [setTag](https://developer.android.com/reference/android/view/View.html#setTag(java.lang.Object))
+     * method.  Very few testing frameworks support looking up
+     * a View by tag, let alone `uiautomatorviewer` which only supports `content-description`, `resource-id`, `text`, and `XPath`.
+     *
+     * While `react-native` does *not* utilize XML based layouts for android Views it
+     * is still possible to support [android:id](https://developer.android.com/reference/android/view/View.html#attr_android:id)
```

Comment:
to support android:id to the underlying view.. Is it possible you meant to use a different word instead of 'support'?